<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEniCS_Blood_Sim</class>
 <widget class="QMainWindow" name="FEniCS_Blood_Sim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEniCS Blood Sim - Graphical Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionRecent_Projects"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit_MainWindow"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionProject_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionImage_Dataset"/>
    <addaction name="separator"/>
    <addaction name="actionFEniCS_Simulation"/>
    <addaction name="separator"/>
    <addaction name="actionPostProcess"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_FEniCs_Blood_Sim"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionExit_MainWindow"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="fenics_blood_sim.qrc">
     <normaloff>:/menu/GUI_Module/images/menu/page_white.png</normaloff>:/menu/GUI_Module/images/menu/page_white.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="fenics_blood_sim.qrc">
     <normaloff>:/menu/GUI_Module/images/menu/folder.png</normaloff>:/menu/GUI_Module/images/menu/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRecent_Projects">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Recent Projects</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="fenics_blood_sim.qrc">
     <normaloff>:/menu/GUI_Module/images/menu/disk.png</normaloff>:/menu/GUI_Module/images/menu/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="fenics_blood_sim.qrc">
     <normaloff>:/menu/GUI_Module/images/menu/save_as.png</normaloff>:/menu/GUI_Module/images/menu/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_MainWindow">
   <property name="icon">
    <iconset resource="fenics_blood_sim.qrc">
     <normaloff>:/menu/GUI_Module/images/menu/door_out.png</normaloff>:/menu/GUI_Module/images/menu/door_out.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="fenics_blood_sim.qrc">
     <normaloff>:/menu/GUI_Module/images/menu/interface_preferences2.png</normaloff>:/menu/GUI_Module/images/menu/interface_preferences2.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionProject_Preferences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Project Preferences</string>
   </property>
  </action>
  <action name="actionImage_Dataset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image Dataset</string>
   </property>
  </action>
  <action name="actionFEniCS_Simulation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>FEniCS Simulation</string>
   </property>
  </action>
  <action name="actionPostProcess">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PostProcess</string>
   </property>
  </action>
  <action name="actionAbout_FEniCs_Blood_Sim">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About FEniCs Blood Sim</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="fenics_blood_sim.qrc">
     <normaloff>:/menu/GUI_Module/images/menu/help.png</normaloff>:/menu/GUI_Module/images/menu/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="fenics_blood_sim.qrc"/>
 </resources>
 <connections/>
</ui>
